git-annex-remote-googledrive

```diff
@@ -152,7 +152,14 @@ class GoogleRemote(ExportRemote):
     @remotemethod
     def initremote(self):
         self._send_version()
+        # - prefix is the name of the folder that will be created to store
+        # the annex.
+        # - root_id will be the gdrive id of this folder when created.
         self.prefix = self.annex.getconfig('prefix')
+        # - parent_folder_id is the gdrive id of the folder where folder 'prefix' will be created
+        #   if undefined, folder 'prefix' folder will be created at the root of
+        #   the user's drive.
+        self.parent_folder_id = self.annex.getconfig('parent_folder_id')
```

Comment:
I don't think we need this to be an object attribute. Once we calculated our root folder, we don't need this anymore, don't you think?